input_Anastasia.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+echo "Welcome, Anastasia"
+
```

Comment:
Well not so simple. You need to use variables to store the name the user inputs...A bit more complex coding...